### PR TITLE
Conductrics intro

### PR DESCRIPTION
**Pull request by 33sticks**

### CODE REVIEW

File: testing/conductrics_setup_abtest_intro.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+
+layout: responsive
+title: conductrics: Intro to Page Editing for Simple A-B Testing
+
+---
+# conductrics: Intro to Page Editing for Simple A-B Testing
```
Comment (randyzwitch) [inline testing/conductrics_setup_abtest_intro.md]: Is conductrics lowercased as a proper noun?

**Comment by randyzwitch** [comment]
Also, need to add link to sidebar

**Comment by 33sticks** [comment]
Good question, I've seen it both ways but I'll go with Caps for now. 

Also, I thought the theme had been updated to auto populate the sidebar, so I'll go ahead and add this change as well.

**Comment by 33sticks** [comment]
I've made the updates, let me know if you need anything else.

**Comment by randyzwitch** [comment]
As a head-up (I didn't know this either), it the header section you can't use colon, it has to be a dash in the title. Don't know why, or if you can escape it instead. I already fixed it.

**Pull request merged**